ds701_book/jupyter_notebooks/19-Regression-III-More-Linear.ipynb

```diff
@@ -7,39 +7,27 @@
         "---\n",
```

Comment:
<p>I find the description of Patsy confusing. Perhaps we should quote directly from patsy package "<code><a href="https://patsy.readthedocs.io/en/latest/API-reference.html#module-patsy" target="_blank">patsy</a></code>&nbsp;is a Python package for describing statistical models (especially linear models, or models that have a linear component) and building design matrices. It is closely inspired by and compatible with the&nbsp;<a href="http://cran.r-project.org/doc/manuals/R-intro.html#Formulae-for-statistical-models" target="_blank">formula</a>&nbsp;mini-language used in&nbsp;<a href="http://www.r-project.org/" target="_blank">R</a>&nbsp;and&nbsp;<a href="https://secure.wikimedia.org/wikipedia/en/wiki/S_programming_language" target="_blank">S</a>."</p><p>From: <a href="https://patsy.readthedocs.io/en/latest/overview.html" target="_blank">https://patsy.readthedocs.io/en/latest/overview.html</a></p><p>See also <a href="https://www.statsmodels.org/stable/example_formulas.html" target="_blank">https://www.statsmodels.org/stable/example_formulas.html</a></p><p> </p><p>Also, should we introduce patsy where we first use it above? And maybe explain a little more about the syntax? This section feels incomplete -- i know it is from the original notes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Reworked the section to include these remarks (see above as well).</p>